LongWarOfTheChosen/Src/LW_Overhaul/Classes/XComGameState_LWOutpost.uc

```diff
@@ -265,9 +269,12 @@ function StateObjectReference CreateRebel(XComGameState NewGameState, XComGameSt
 			{
 				FacelessChance *= 0.6;
 			}
-			if (Unit.HasSoldierAbility('MindMerge'))
+			foreach default.FacelessReductionPsiAbilities(PsionAbilityName)
```

Comment:
This time the multiplier will trigger for EACH of those abilities. Use find instead of the loop.